relationships/synthesis/APM-APPLICATION-to-DATABASE.yml

```
@@ -0,0 +1,20 @@
+relationships:
+  - name: apmCallsDatabase
+    version: 1
+    origins:
+      - APM Metrics
+    conditions:
+      - attribute: metricName
+        regex: "^datastore/instance/[^/]*/([^/]*\\.rds\\.amazonaws\\.com[^/]*)/.*"
```

I just noticed this is missing a parenthesis (based on the original)
Given we are not using or supporting capture groups guess we have one that is not needed 

```suggestion
        regex: "^datastore/instance/[^/]*/[^/]*\\.rds\\.amazonaws\\.com[^/]*)/.*"
```